index.html

```diff
@@ -4,7 +4,7 @@
 <head>
   <meta charset="UTF-8">
   <title>Tribute Page</title>
-  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="style.css">
```

Comment:
- [ ] Good Job for using the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) But I recommend you to **phrased the comments politely, as suggestions instead of orders** and  **always includes an explanation, a "why", for each requested change**. For Example : -
- " Kindly I recommend you to  link the style-sheet correctly, where the file is originally located. it makes to give your site a more engaging look across devices" .